Source/Phone/Call/CallManager.swift

```diff
@@ -62,9 +62,10 @@ class CallManager {
         return nil
     }
     
-    func fetchActiveCalls() {
+    func fetchActiveCalls(deviceRegistrationInformation: DeviceRegistrationInformation) {
```

Comment:
What's the reason we need to take the deviceRegistrationInformation parameter here? The CallManager already holds the deviceService.